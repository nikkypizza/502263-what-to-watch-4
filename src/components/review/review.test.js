import React from 'react';
import renderer from 'react-test-renderer';

import Review from './review.jsx';
import promoObj from '../../mocks/promo.js';

describe(`Review Component`, () => {
  it(`renders`, () => {
    const tree = renderer.create(
        <Review
          dataObj={promoObj}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
