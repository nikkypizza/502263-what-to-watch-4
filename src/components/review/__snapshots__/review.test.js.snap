// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Component renders 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    />
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      />
      <time
        className="review__date"
      >
        Invalid Date
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8.9
  </div>
</div>
`;
